app.js

```diff
@@ -23,13 +32,13 @@ CookieStore.prototype.randomCustomers = function(){
 };
 
 //This is a function that can be called for each store/object.
-var getCookiesPerHour = function(objectName){
+function getCookiesPerHour(objectName){
```

Comment:
Great! But, can you think of a way to make this a method instead of a free-floating function?